pkg/router/f5/f5.go

```diff
@@ -459,6 +459,35 @@ func (f5 *f5LTM) delete(url string, result interface{}) error {
 }
 
 //
+// iControl REST resource helper methods.
+//
+
+// iControlResourceUriPath returns a fully qualified `escaped` resource URI path component that can be used in the iControl REST calls.
+func (f5 *f5LTM) iControlResourceUriPath(resourceName string) string {
+	resourcePath := path.Join(f5.partitionPath, resourceName)
+	return escapeiControlUriPath(resourcePath)
+}
+
+// iControlVserverUriPath returns a fully qualified `escaped` Virtual Server path that can be used in the iControl REST calls.
+func (f5 *f5LTM) iControlVserverUriPath(vserverName string) string {
+	// Build the fully qualified Virtual Server name.
+	// Note: Although the configuration is stored inside a partition path, the F5 virtual server
+	// belongs to the partition's top-level folder (directory).
```

Comment:
If it doesn't introduce any inaccuracy, I believe it would be clearer to say, "Most resources are stored under the configured partition, which may be a top-level folder or a sub-folder.  However, vservers can only be stored under top-level folders."

The line wrapping is weird.